src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -150,7 +150,14 @@ public string LogDirectory
 
         public string DefaultPackagesDirectory
         {
-            get { return TransformPath(RootDirectories.First(), PackagesDirectoryName); }
+            get
+            {
+                if (Preferences is PreferenceSettings preferences)
```

Comment:
I'm just curious about this, are these two different overloads of TransformPath?

Reply:
oh, I see now, Preferences is an IPreferences and you added the `SelectedPackagePathForInstall` to the concrete class.
👍

Reply:
do we have other types of implementations of IPreferences ?
If not...I wonder why `SelectedPackagePathForInstall` is not part of the Interface...

Reply:
We cannot change the interface right now. In the future yes, we should be adding it to the interface.